app/src/main/java/com/babylon/wallet/android/presentation/account/AccountViewModel.kt

```diff
@@ -212,7 +213,7 @@ class AccountViewModel @Inject constructor(
             _state.update { state -> state.copy(pendingStakeUnits = true) }
             viewModelScope.launch {
                 updateLSUsInfo(account, stakes).onSuccess {
-                    _state.update { state -> state.copy(pendingStakeUnits = false) }
+                    _state.update { state -> state.onValidatorsReceived(it) }
```

Comment:
there is no way to modify db query so that it fires on the update instead of manual update?